code/modules/admin/admin_verbs.dm

```diff
@@ -555,6 +555,27 @@ ADMIN_VERB_ADD(/client/proc/man_up, R_ADMIN, FALSE)
 	log_admin("[key_name(usr)] told [key_name(T)] to man up and deal with it.")
 	message_admins("\blue [key_name_admin(usr)] told [key_name(T)] to man up and deal with it.", 1)
 
+ADMIN_VERB_ADD(/client/proc/perkadd, R_ADMIN, FALSE)
+/client/proc/perkadd(mob/T as mob in SSmobs.mob_list)
+	set category = "Fun"
+	set name = "Add Perk"
+	set desc = "Add a perk to a person."
+	var/datum/perk/perkname = input("What perk do you want to add?") in subtypesof(/datum/perk/)
+	T.stats.addPerk(perkname)
```

Comment:
As `input()` sleeps, you need to check that T wasn't QDELETED or marked for QDEL in the meantime. Use the defined : `QDELETED(var)` to check for it.